EU4ToVic2/Source/V2World/Country/CountryDetails.cpp

```diff
@@ -3,30 +3,35 @@
 #include "OSCompatibilityLayer.h"
 #include "ParserHelpers.h"
 
-V2::CountryDetails::CountryDetails(std::string _filename): filename(std::move(_filename))
+V2::CountryDetails::CountryDetails(std::filesystem::path _filename): filename(std::move(_filename))
 {
 	registerKeys();
 
-	if (commonItems::DoesFileExist("./blankMod/output/common/countries/" + filename))
+	if (commonItems::DoesFileExist(std::filesystem::path("blankMod/output/common/countries/") / filename))
```

Comment:
Since filename is already a path, you should be able to skip converting the initial string:
```suggestion
	if (commonItems::DoesFileExist("blankMod/output/common/countries/" / filename))
```